src/components/Board.js

```diff
@@ -16,17 +17,97 @@ class Board extends Component {
     };
   }
 
+  componentDidMount() {
+    axios.get(CARDS_URL)
+    .then((response) => {
+      console.log('Success!');
+
+      this.setState({cards: response.data })
+      this.props.updateStatusCallback(`Loaded all cards successfully`)
+
+    })
+    .catch((error)=>{
+      console.log('Error :(');
+      this.props.updateStatusCallback(`Error unable to load cards`)
+
+    });
+  }
+
+
+  cardClicked = (index, id) => {
+
+    const updatedCard = this.state.cards;
+       updatedCard.splice(index, 1 );
+      // updatedCard.pop(deletedCard)
+       console.log(updatedCard.splice(index, 1 ));
+
+      this.setState({cards: updatedCard});
```

Comment:
Be careful about updating state before the api call finishes.  What if it can't?  Then the user might THINK the card was deleted when it wasn't.